### DISCUSSION TITLE
SamGEO with Streamlit APP

### DISCUSSION THREAD

**Discussion by agtech223** [opengeos/segment-geospatial#376] [Q&A] [answered]
Just wondering, is anyone implemented SAMGeo with Streamlit app? I am not able to display map with these lines below:
m = sam.show_map(zoom=15)
m.to_streamlit(height=700)

is there any workaround? I am able to display tiff images on strramlit using Leafmap library. But SAM is not working on streamlit.

**Comment by giswqs** [comment, marked as answer]
streamlit does not support ipyleaflet. You won't be able to use the samgeo interactive segmentation with streamlit. Don't waste you time on this.

**Comment by agtech223** [comment]
Thanks for the prompt response. Is there any other alternative website option for samgeo?

**Reply by giswqs** [reply]
Try https://solara.dev/

**Reply by agtech223** [reply]
Thanks for suggestion and your amazing work for open source community!
